src/runtime/windows_clock.cpp

```diff
@@ -35,7 +35,7 @@ WEAK int64_t halide_current_time_ns(void *user_context) {
 }
 
 WEAK void halide_sleep_ms(void *user_context, int ms) {
-	Sleep(ms);
+        Sleep(ms);
```

Comment:
Looks like I introduced these tabs via msvc, and this PR changes them to 8 spaces. Mind doing a drive-by conversion to 4 spaces in this PR? (to avoid the merge conflict if I fix in master)